backend/dal/Services/Admin/Concrete/UserService.cs

```diff
@@ -237,6 +237,42 @@ public Paged<AccessRequest> GetAccessRequests(int page = 1, int quantity = 10, s
             var accessRequests = query.Skip((page - 1) * quantity).Take(quantity);
             return new Paged<AccessRequest>(accessRequests, page, quantity, query.Count());
         }
+
+        /// <summary>
+        /// Get all the access requests that users have submitted to the system
```

Comment:
Summary doesn't represent what the function actually does.
`Get all access requests that match the specified 'filter'.`